packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -84,7 +84,7 @@ enum _RefreshIndicatorMode {
 class RefreshIndicator extends StatefulWidget {
   /// Creates a refresh indicator.
   ///
-  /// The [onRefresh], [child], and [notificationPredicate] arguments must be
+  /// The [onRefresh], [child], [edgeDistance], and [notificationPredicate] arguments must be
```

Comment:
Hi, as per the code, wouldn't edgeDistance have a default value, instead of being required a non-null value?

Reply:
Yup, indeed. Submitting a patch.